code/modules/mechs/mech_interaction.dm

```diff
@@ -63,17 +63,24 @@
 	if(modifiers["shift"])
 		user.examinate(A)
 		return
+		
+	if(modifiers["ctrl"])
+		if(selected_system)
+			if(selected_system == A)
+				selected_system.CtrlClick(user)
+				setClickCooldown(3)
+			return	
```

Comment:
Feels like this is in the wrong scope, probably because of the obnoxious way the checks above are worded. I would do
```
if(modifiers["ctrl"] && selected_system && selected_system == A)
	do_stuff
	return
```
unless you want that specific level of ctrl-preemption.

Reply:
Collapsed it into 
```
if(modifiers["ctrl"] && selected_system == A)
	do_stuff
	return
```
since p sure you can't click on null so dun need that check